labex/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Switch, Route, BrowserRouter } from "react-router-dom";
+
+import HomePage from './components/HomePage';
+import ListTripsPage from './components/ListTripsPage';
+import FormPage from './components/FormPage';
+
+import LoginPage from './components/LoginPage';
+import AdmPage from './components/AdmPage';
+import AdmCreateTripPage from './components/AdmCreateTripPage';
+import AdmDetailsPage from './components/AdmDetailsPage';
+import AdmTripsPage from './components/AdmTripsPage';
+
+const App = () => {
+  return (
+    <BrowserRouter>
+        <Switch>
+          <Route exact path="/">
+            <HomePage />
+          </Route>
```

Comment:
É possível reduzir essa estrutura utilizando a propriedade **_component_** dentro do **_Route_**.
```suggestion
          <Route exact path="/" component={HomePage} />
```
Isso também vale para as outras rotas.

Reply:
Boa, Yuzo! Vou aplicar nas proximas aplicações!